tokio-console/src/config.rs

```diff
@@ -95,6 +96,18 @@ pub enum OptionalCmd {
     ///     $ tokio-console gen-config > console.toml
     ///
     GenConfig,
+
+    /// Generate shell completions
+    ///
+    /// The completion script will be written to stdout.
+    /// The completion script should be saved in the shell's completion directory.
+    /// This depends on which shell is in use.
+    GenCompletion {
+        #[clap(name = "install", long = "install")]
+        install: bool,
+        #[clap(arg_enum)]
+        shell: Shell,
```

Comment:
i wonder if it's worthwhile to also try to read this from the value of the `$SHELL` environment variable, if it's not provided by the user? we could add that in a follow-up.